scripts/automation-generate-and-commit-go-sdk.sh

```diff
@@ -86,14 +92,22 @@ function conditionallyCommitAndPushGoSdk {
     git add --all
     git commit -m "Updating based on $sha"
 
+    # Microsoft Graph: conditional update of dependencies
+    cd microsoft-graph/
+    go mod tidy
+    if [[ $(git status --porcelain | wc -l) -gt 0 ]]; then
+      git add --all
+      git commit -m "microsoft-graph: updating dependencies based on $sha"
+    fi
+    cd "$workingDirectory"
```

Comment:
This may need temporarily disabling/reverting, will know for sure when Phase 6 is completed.